BigBaseV2/src/core/globals.hpp

```diff
@@ -284,6 +294,8 @@ namespace big
 				{
 					"self", {
 						{ "godmode", this->self.godmode },
+						{ "invisibility", this->self.invisibility },
+						{ "clean_player", this->self.clean_player },
```

Comment:
Set these alphabetically, to follow the same coding style as the entire project.